test/test_podman_baseline.sh

```diff
@@ -20,24 +21,27 @@
 #######
 showerror=0
 installdocker=0
-while getopts "de" opt; do
+usedocker=1
+while getopts "den" opt; do
     case "$opt" in
     d) installdocker=1
        ;;
     e) showerror=1
        ;;
+    n) usedocker=0
+       ;;
     esac
 done
 
-if [ "$installdocker" -eq 1 ]
+if [ "$installdocker" -eq 1 ] && [ "usedocker" -ne 0 ]
 then
     echo "Script will install and then deinstall Docker."
 fi
 
 if [ "$showerror" -eq 1 ]
 then
     echo "Script will stop on unexpected errors."
-    set -eu
+    set -e
```

Comment:
Probably doesn't matter, but wondering if you ran into an issue with the -u here?  It should just be checking for unset params and variables.

Reply:
Yeah, it does trigger a problem when combined with -e.